apps/journeys-admin/src/components/Editor/Slider/Content/Canvas/QuickControls/DeleteBlock/DeleteBlock.tsx

```diff
@@ -45,6 +49,7 @@ export function DeleteBlock({
   const handleOpenDialog = (): void => setOpenDialog(true)
   const handleCloseDialog = (): void => {
     setOpenDialog(false)
+    setOpenDeleteDialog?.(false)
```

Comment:
```suggestion
    setOpenDeleteDialog(false)
```

since we will be making this prop required